### DISCUSSION TITLE
添加HTTP GET型代理

### DISCUSSION THREAD

**Discussion by uncleheart** [v2fly/v2ray-core#507] [General]
场景：
需要使用海外动态IP代理。使用v2ray从国内到国外转发。

问题：
大部分http代理不支持connect连接非443端口网站，从而导致http网站无法使用。
例如squid就默认不支持connect

**Comment by kslr** [comment]
听不懂你说什么，你可以自己写一个http隧道

**Reply by uncleheart** [reply]
是需要使用外部的http代理。但外部http在明文http场景下，不允许http隧道。

**Comment by uncleheart** [comment]
链路：
国内->国外->外部http代理
链路举例：
国内：In http  , out vless
国外：in vless， out http

问题点：
访问 http://baidu.com 。到了国外时，
v2ray会使用connect建立http tunner
connect baidu.com:80 HTTP/1.1
但是外部http代理发现是connect，但是连接的是非443端口，直接报错。（这个配置是正确的，都默认如此配置）
